### ISSUE TITLE
Strange behavior after update

### ISSUE THREAD

**Issue by cjvanlissa** [Vimux/Mainroad#186]
Dear Vimux, after installing the updated Mainroad template, my website renders in an unexpected way: Menu items are pushed downward, the sidebar is at the bottom of the page, and social icons are rendered black on black, rather than white on black. Any idea what could be causing this?
THank you sincerely,
Caspar 

The link: [http://developmentaldatascience.org/](http://developmentaldatascience.org/)
Repository: https://github.com/cjvanlissa/developmentaldatascience

**Comment by Vimux** [comment]
Hello,

> http://developmentaldatascience.org/

Looks OK at first glance. Screenshot?

> https://github.com/cjvanlissa/developmentaldatascience

Private repository, I guess.

---

> Any idea what could be causing this?

Hugo upgrade, additional theme customization, old site config revision?

Сould you please point:

* Your Hugo version
* Old commit hash of theme before update
* New commit hash of theme

**Comment by cjvanlissa** [comment]
Thank you for the swift response; I'm not sure I can provide you with all the information requested, but I'll share what I can. A few hours ago, I updated to Hugo 0.68.3, and then I updated Mainroad, so that would be commit 
`ccff77d`. I don't know which version of Mainroad I was on before, unfortunately. 

Here is a screenshot of the live website viewed from my device. I made a few cuts to fit all interesting aspects into one picture, and highlighted them in green: https://imgur.com/TotPpx8

I also made my repository public.

I customized the following things:

1) Add Disqus to `comments.html`
2) Add an image to `header.html`
3) Add RSS feed to `social.html` widgets
4) Add RSS.xml
